x-pack/plugin/esql/qa/testFixtures/src/main/resources/drop.csv-spec

```diff
@@ -49,29 +49,136 @@ b:integer | x:integer
 ;
 
 dropAllColumns
+required_capability: fix_no_columns
 from employees | keep height | drop height | eval x = 1;
```

Comment:
Can we add a limit after the from here? It's a bit weird otherwise 😆 (Or switch to a smaller index)

Reply:
I wanted to keep the original test, but yeah, we can add a limit 😄